README.md

```diff
@@ -1,6 +1,10 @@
 Commissaire
 ===========
-The lightweight REST API for Node Management.
+Commissaire is a lightweight REST interface for performing system management tasks on network hosts in a cluster through Ansible.
+
+Current capabilities include rolling upgrades and restarts of traditional or "[Atomic](http://projectatomic.io)" hosts, and bootstrapping new hosts into an existing "container management" system such as [OpenShift](https://www.openshift.com) or [Kubernetes](https://kubernetes.io).
+
+Moving forward, Commissaire will expand the scope of its REST interface to provide centralized host inventory management and consoldate various RHEL and atomic host capabilities into a centralized API.
```

Comment:
I'd take "RHEL" out of here, this is the upsteam project.  Alternately, you can use "RHEL, CentOS or Fedora".  But even that limits you; what if Debian was interested in Commissaire?